app/views/layouts/application.html.haml

```diff
@@ -19,6 +19,9 @@
               = link_to "Invoices", ""
             %li{class: "#{top_navbar_active_class_name('clients','index')}"} 
               = link_to "Clients", clients_path
+            -if current_user.present? && current_user.role.to_s == "Admin"
+              %li{class: "#{top_navbar_active_class_name('companies','new')}"} 
+                = link_to('Companies',"#modal-form","data-toggle" => "modal")
           %ul.nav.navbar-nav.pull-right
             -if !current_user.blank?
```

Comment:
This is bad style again

Avoid using `!something` if possible

check http://api.rubyonrails.org/classes/Object.html#method-i-present-3F